contrib/charts/navigator/values.yaml

```diff
@@ -25,6 +25,15 @@ apiserver:
     tag: latest
     pullPolicy: Always
   logLevel: 2
+
+  persistence:
+    enabled: false
+    size: 10Gi
+    accessMode: ReadWriteOnce
```

Comment:
Why is accessMode configurable?  `ReadWriteOnce` seems like the only sensible option.